<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>StartGameMenu</class>
 <widget class="QWidget" name="StartGameMenu">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1200</width>
    <height>800</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QPushButton" name="pushButtonBackstart">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>720</y>
     <width>120</width>
     <height>40</height>
    </rect>
   </property>
   <property name="palette">
    <palette>
     <active>
      <colorrole role="ButtonText">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>0</red>
         <green>0</green>
         <blue>0</blue>
        </color>
       </brush>
      </colorrole>
     </active>
     <inactive>
      <colorrole role="ButtonText">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>0</red>
         <green>0</green>
         <blue>0</blue>
        </color>
       </brush>
      </colorrole>
     </inactive>
     <disabled>
      <colorrole role="ButtonText">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>148</red>
         <green>148</green>
         <blue>148</blue>
        </color>
       </brush>
      </colorrole>
     </disabled>
    </palette>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Back to menu</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonBGMstart">
   <property name="geometry">
    <rect>
     <x>1100</x>
     <y>30</y>
     <width>81</width>
     <height>32</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="BGMTextstart">
   <property name="geometry">
    <rect>
     <x>1070</x>
     <y>10</y>
     <width>58</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="text">
    <string>BGM</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonSingle">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>210</y>
     <width>300</width>
     <height>100</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Phosphate</family>
     <pointsize>24</pointsize>
     <weight>50</weight>
     <italic>false</italic>
     <bold>false</bold>
     <underline>false</underline>
     <strikeout>false</strikeout>
     <kerning>true</kerning>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>ArrowCursor</cursorShape>
   </property>
   <property name="focusPolicy">
    <enum>Qt::TabFocus</enum>
   </property>
   <property name="contextMenuPolicy">
    <enum>Qt::DefaultContextMenu</enum>
   </property>
   <property name="text">
    <string>Single Players</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonMulti">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>590</y>
     <width>300</width>
     <height>100</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Phosphate</family>
     <pointsize>24</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Multi players</string>
   </property>
  </widget>
  <widget class="QLabel" name="gametitle">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>50</y>
     <width>441</width>
     <height>71</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>DIN Alternate</family>
     <pointsize>50</pointsize>
     <weight>50</weight>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="text">
    <string>Choose Play Mode</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonDouble">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>400</y>
     <width>300</width>
     <height>100</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Phosphate</family>
     <pointsize>24</pointsize>
     <weight>50</weight>
     <italic>false</italic>
     <bold>false</bold>
     <underline>false</underline>
     <strikeout>false</strikeout>
     <kerning>true</kerning>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>ArrowCursor</cursorShape>
   </property>
   <property name="focusPolicy">
    <enum>Qt::TabFocus</enum>
   </property>
   <property name="contextMenuPolicy">
    <enum>Qt::DefaultContextMenu</enum>
   </property>
   <property name="text">
    <string>Double Players</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
